gamemodes/zombieplague/gamemode/sv_roundmanager.lua

```diff
@@ -120,9 +120,30 @@ function RoundManager:Prepare()
 	end
 	return false
 end
+
 function RoundManager:TryNewRound()
 	if RoundManager:Prepare() then
-		RoundManager:SetTimer(cvars.Number("zp_infection_delay",  10), RoundManager.NewRound)
+		RoundManager:SetTimer(cvars.Number("zp_infection_delay",  13), RoundManager.NewRound)
+		for i, ply in ipairs(RoundManager:GetPlayersToPlay()) do
```

Comment:
Should this not be configurable? I mean, some player might want to have the option of not be freezed